fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/AbstractResourceIT.java

```diff
@@ -111,7 +111,16 @@ public void setLogger() {
     protected static CloseableHttpClient client = createClient();
 
     protected static CloseableHttpClient createClient() {
-        return HttpClientBuilder.create().setMaxConnPerRoute(MAX_VALUE).setMaxConnTotal(MAX_VALUE).build();
+        return createClient(false);
+    }
+
+    protected static CloseableHttpClient createClient(final boolean disableRedirects) {
+        if (disableRedirects) {
```

Comment:
I think it would be cleaner to write method body with one return statement and no duplication of the create(), setMaxConnPerRoute(), setMaxConnTotal(), and build() statements.

Reply:
But that may be a question of style. So your call.